xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-ui/src/main/resources/XWiki/SolrSearchAdminIndexingUserSheet.xml

```diff
@@ -0,0 +1,324 @@
+<?xml version="1.1" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<xwikidoc version="1.5" reference="XWiki.SolrSearchAdminIndexingUserSheet" locale="">
+  <web>XWiki</web>
+  <name>SolrSearchAdminIndexingUserSheet</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <parent>Main.WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <version>1.1</version>
+  <title>SolrSearchAdminIndexingUserSheet</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>true</hidden>
+  <content>== {{translation key="admin.indexinguser" /}} ==
+
+(% class="noitems" %)
+{{translation key="admin.indexinguser.explanation.part1" /}}
+
+(% class="noitems" %)
+{{translation key="admin.indexinguser.explanation.part2" /}}
```

Comment:
This should maybe be highlighted a bit more (in a warning macro maybe) as people will be tempted to put the main wiki admin.

Reply:
Good idea. I have updated the UI to put the security implications inside a warning box (and also removes the `class="noitems"` form the rest of the text as it looked too small compared to the box.